style.css

```diff
@@ -875,3 +880,510 @@ ul {
   font-size: 18px;
   color: #fff;
 }
+
+/* Desktop version */
+@media screen and (min-width: 768px) {
+  body {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    overflow-x: hidden;
+    background-color: #e5e5e5;
+  }
+
+  .logo {
+    font-size: 25px;
+    font-weight: 700;
+    line-height: 20px;
+    background-color: transparent;
+    margin-left: 10%;
+    margin-top: 2%;
+  }
+
+  .app-header {
+    display: flex;
+  }
+
+  .humbourger {
+    display: none;
+  }
+
+  .ambony {
+    width: auto;
+    margin-right: 5%;
+  }
+
+  .menu {
+    display: flex !important;
+    justify-content: flex-end;
+    align-items: center;
+  }
+
+  .menu-item {
+    margin-right: 20px;
+  }
+
+  .menu-item a {
+    display: flex;
+    font-size: 1rem;
+    color: #344563;
+    text-decoration: none;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 20px;
+    text-align: center;
+  }
+
+  .header-ambony {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    width: 100%;
+    padding-left: 10%;
+    background-color: #fff;
+    background-image: url(media/Header\ bg.svg);
+    background-size: cover;
+    top: 2.875rem;
+    border-bottom-left-radius: 60px;
+    height: 44.88rem;
+  }
+
+  .header-container {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    padding-left: 10%;
+  }
+
+  .narson {
+    width: 100%;
+  }
+
+  .detail {
+    height: 98px;
+    width: 920px;
+    left: 0;
+    top: 134px;
+  }
+
+  .connection {
+    margin-top: -1px;
+    font-size: 16px;
+    font-weight: 500;
+    line-height: 24px;
+    text-align: left;
+    color: #7f8cff;
+  }
+
+  /* work section */
+
+  .work {
```

Comment:
- [ ] I would suggest that you stick to the design as much as possible (e.g., font, colors, images, text, margins). Align the content of each card to look like the original design.
- [ ] The buttons on your cards do not align with the provisions of the original design. Here is a guide provided in Figma  
- [ ] Also apply the appropriate margins and paddings to the buttons and content of the card so that each card looks exactly like the original design.
- [ ] Kindly make these changes on all the cards.

> Figma Design
<img width="594" alt="Screenshot 2023-05-04 at 16 09 54" src="https://user-images.githubusercontent.com/71140133/236262439-ed9e9902-e09c-4270-8545-cd4982d90fd3.png">

> Your design
<img width="1089" alt="Screenshot 2023-05-04 at 16 07 40" src="https://user-images.githubusercontent.com/71140133/236262700-ab8b75ac-7640-4c09-b529-a14d0cd737ad.png">